### PR TITLE
Export 5-day workshop data to S3 via cron

### PR DESCRIPTION
**Pull request by bencodeorg**
Automates export of PD workshops to S3 for outreach team to use in tracking upcoming summer workshops.

Check out in particular odd use of a throwaway class to gain access to the Api::CsvDownload module -- might be a better way to do this, but best way I could find to gain access to the `generate_csv` method.

### CODE REVIEW

File: bin/cron/workshops_to_s3 [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# in the workshop dashboard.
+def main
+  workshops = Pd::Workshop.
+    where(
+      {
+        course: [Pd::Workshop::COURSE_CSP, Pd::Workshop::COURSE_CSD],
+        subject: Pd::Workshop::SUBJECT_SUMMER_WORKSHOP
+      }
+    ).
+    scheduled_start_on_or_after(Date.new(2019, 5, 1))
```
Comment (clareconstantine) [inline bin/cron/workshops_to_s3]: Not super important, but I think I would have styled this like this based on our conventions:
```workshops = Pd::Workshop.
    where({
        course: [Pd::Workshop::COURSE_CSP, Pd::Workshop::COURSE_CSD],
        subject: Pd::Workshop::SUBJECT_SUMMER_WORKSHOP
      }).scheduled_start_on_or_after(Date.new(2019, 5, 1))
```

File: bin/cron/workshops_to_s3 [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  workshop_hash = workshops.map {|w| Api::V1::Pd::WorkshopDownloadSerializer.new(w).attributes}
+
+  # Generate throwaway class that has functionality
+  # from Api::CsvDownload module to emulate what happens in
+  # controller/api/vi/pd/workshop_controller's filter method
+  output = Class.new.extend(Api::CsvDownload).generate_csv workshop_hash
```
Comment (clareconstantine) [inline bin/cron/workshops_to_s3]: I think you should be able to call `include Api::CsvDownload` inside of `main` and then use `generate_csv` [(example)](https://github.com/code-dot-org/code-dot-org/blob/staging/bin/cron/calculate_workshop_survey_results)

Comment (bencodeorg) [reply bin/cron/workshops_to_s3]: Updated to use `extend` per offline conversation.

File: bin/cron/workshops_to_s3 (hunk 1 above)
Comment (bencodeorg) [reply bin/cron/workshops_to_s3]: Updated to meet linter standards.

**Review by clareconstantine — APPROVED** [review]
woo!

**Pull request merged**